### DISCUSSION TITLE
Erro ao buscar imagens blob do curso: Angular 8 e UIKIT

### DISCUSSION THREAD

**Discussion by gabrielrcdl** [balta-io/forum#313] [Tira Dúvidas] [answered]
Olá alguém passou por esse problema?  curso 7187

<img width="943" alt="image" src="https://user-images.githubusercontent.com/78031815/232791087-a63ecbee-5ba6-4cd5-bc14-c0a83d19fee7.png">

**Comment by andrebaltieri** [comment, marked as answer]
@gabrielrcdl acredito que deva ser um problema no nosso storage, poderia utilizar algum placeholder no lugar da imagem do storage?
https://placeholder.com
